internal/controller/streams.go

```diff
@@ -203,15 +206,48 @@ func (sm *streamManager) StreamCiliumNetworkFlows(ctx context.Context, ciliumNam
 			err = ciliumFlowCollector.exportCiliumFlows(ctx, *sm)
 			if err != nil {
 				sm.logger.Warnw("Failed to collect and export flows from Cilium Hubble Relay", "error", err)
-				sm.streamClient.disableNetworkFlows = true
+				sm.streamClient.disableNetworkFlowsCilium = true
 				return err
 			}
 		}
 	}
 	return nil
 }
 
-// connectAndStreamCiliumNetworkFlows creates ciliumNetworkFlows client and begins the streaming of network flows.
+// StreamFalcoNetworkFlows handles the falco network flow stream.
+func (sm *streamManager) StreamFalcoNetworkFlows(ctx context.Context) error {
+	var falcoFlow string
+	for {
+		falcoFlow = <-sm.streamClient.falcoEventChan
+		if filterIllumioTraffic(falcoFlow) {
+			// Extract the relevant part of the output string
+			re := regexp.MustCompile(`\((.*?)\)`)
```

Comment:
Move this into a global variable, and call the `MustCompile` code into `func init() {...}` so this is compiled only once, as this call is expensive.